activiti-cloud-services-org/activiti-cloud-services-org-service/src/main/java/org/activiti/cloud/services/organization/service/ModelService.java

```diff
@@ -236,37 +339,50 @@ public Model importModel(Project project,
                 fileContent.getFilename(),
                 fileContent));
 
-        Model model = importModelFromContent(project,
-                                             modelType,
-                                             fileContent);
+        Model model = importModelFromContent(project, modelType, fileContent);
+        return model;
+    }
 
-        return updateModelContent(model,
-                                  fileContent);
+    public Model importModelFromContent(Project project, ModelType modelType, FileContent fileContent) {
+      Model model = null;
+      if(modelTypeService.isJson(modelType) || fileContent.getContentType().equalsIgnoreCase("application/json")){
```

Comment:
just guessing, can you use it `ContentTypeUtils.JSON` instead ?

Reply:
fixed